stand-ups/readme.md

```diff
@@ -1,18 +1,18 @@
 # Stand-ups
 
-The stand-up is a ceremony that is held each day of the sprint. In this ceremony, each contributor will answer three simple questions. This will repeat until each contributor has answered the three following questions.
+The stand-up is a time-box ceremony that is held each day of the sprint. In this ceremony, each contributor in the Development Team will answer three simple questions. This will repeat until each contributor has answered the three following questions.
 
-1. What did you work on yesterday?
-2. What are you working on today?
-3. Do you have any impediments/blockers? (defer discussion / resolution to "the parking lot", described below)
+> 1. What did you work on yesterday that contributes to meet the sprint goal?
+> 2. What are you working on today that will contribute to meet the sprint goal?
+> 3. Do you have any impediments/blockers? (defer discussion / resolution to "the parking lot", described below)
 
 After that point, the stand-up is concluded. There may be a parking lot discussion after. However, participation in the parking lot discussion is optional for all members except those explicitly needed for discussion of the issues raised.
 
 The term parking lot refers to a bucket of comments, concerns, or questions that will be discussed and/or addressed at a later point with potentially fewer contributors. This is part of a strategy to avoid letting the discussion in a meeting shift to a subject that is not aligned with the meeting goals and/or decisions.

```

Comment:
During the standup, Any participant can identify a topic to be discussed after the meeting, tagging it as a follow up item, and adding it to the ‘Parking Lot.’ This means it will be discussed ‘outside’ afterward, rather than right now, during the standup meeting.

Reply:
@suroor-fatima this is a fundamental point to make the daily short and objective. I believe the paragraph "After that point, the stand-up is concluded. There may be a parking lot discussion after. However, participation in the parking lot discussion is optional for all members except those explicitly needed for discussion of the issues raised." covers this point already when it mentions that there may be a parking lot discussion after and the participation is optional to members not directly involved in the item. Please let me know in case I misunderstood your comment.

Reply:
My point was - parking lot discussion topics need to be captured as the stand up is going around (by one person designated - can be the TPM or any one team member) so that folks don't forget at the end or lose context. Also if other team members have questions, they can rest assured that the topic is captured for the parking lot. This process needs to be clear.

Reply:
Got it. Thanks to clarify. I will include this information.